src/query/graphite/native/builtin_functions.go

```diff
@@ -242,6 +243,42 @@ func timeShift(
 	}, nil
 }
 
+
+func timeSlice(ctx *common.Context, inputPath singlePathSpec, start string, end string) (ts.SeriesList, error) {
+	tzOffsetForAbsoluteTime := time.Duration(0)
+	startTime, err := graphite.ParseTime(start, time.Now(), tzOffsetForAbsoluteTime)
+	endTime, err := graphite.ParseTime(end, time.Now(), tzOffsetForAbsoluteTime)
+
+	if (err != nil) {
+		return ts.NewSeriesList(), err
+	}
```

Comment:
Don't need round brackets here when checking err, also missed the first error (since it gets overwritten by the second statement `endTime, err := ...`).

This is more idiomatic Go written like this
```suggestion
	tzOffsetForAbsoluteTime := time.Duration(0)
	startTime, err := graphite.ParseTime(start, time.Now(), tzOffsetForAbsoluteTime)
	if err != nil {
		return ts.NewSeriesList(), err
	}
	endTime, err := graphite.ParseTime(end, time.Now(), tzOffsetForAbsoluteTime)
	if err != nil {
		return ts.NewSeriesList(), err
	}
```